.travis.yml

```diff
@@ -1,8 +1,21 @@
 ---
 language: go
 
+sudo: required
+
 notifications:
   email: false
 
+before_install:
```

Comment:
NIT: I think that steps under `before_install` stanza are not required to generate codecov report.